web/pano_editor.css

```
@@ -1369,25 +1386,99 @@
+  width: 40px;
+  height: 40px;
+  border-radius: calc(var(--pano-float-radius) - var(--pano-float-pad));
+  color: var(--pano-text-secondary);
+  cursor: grab;
+  touch-action: none;
+  user-select: none;
+}
+.pano-frame-roll-knob.dragging { cursor: grabbing; }
+.pano-frame-roll-knob.armed { color: rgb(255, 221, 87); }
+.pano-frame-roll-knob svg { width: 28px; height: 28px; }
+.pano-frame-roll-knob-ring { fill: none; stroke: currentColor; stroke-width: 1; opacity: 0.42; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)